### DISCUSSION TITLE
New 5.0.0 release with fix for issue 2134

### DISCUSSION THREAD

**Discussion by nthakrar** [JasperFx/marten#2142] [General] [answered]
I was wondering if there's a planned release for 5.0.0-x with the fix for [the issue](https://github.com/JasperFx/marten/issues/2134) where the provided connection string is not honoured when opening query sessions? If so, when would that be?

Thanks!

**Comment by nthakrar** [comment]
@VilleHakli any idea? Cheers!

**Comment by mysticmind** [comment]
Most likely in this week, we will push out a release.

**Comment by mysticmind** [comment, marked as answer]
v5 is released a little while ago with the fix for #2134
